src/api.js

```diff
@@ -3,7 +3,7 @@ let debug       = require('debug')('app:api');
 let slugid      = require('slugid');
 let assert      = require('assert');
 let _           = require('lodash');
-let base        = require('taskcluster-base');
+let base        = require('taskcluster-lib-api');
```

Comment:
I noticed in the comments references to base, for example: https://github.com/owlishDeveloper/taskcluster-queue/blob/a2ed15fd0b144ca332396a3b928359d7dbfcafda/src/api.js#L117 - does it mean, if I deleted base, I should require, in this case, pulse-publisher (I saw in tc-base `Exchanges:      'pulse-publisher'`)? Also, there were couple of base.Entity, I changed that as well, but I'm not sure if that's correct...